Rakefile

```diff
@@ -1,23 +1,19 @@
-require "bundler/gem_tasks"
-require 'jars/version'
 
-begin
-  require 'rspec/core/rake_task'
-  RSpec::Core::RakeTask.new(:spec)
-rescue LoadError
-end
-
-task default: "spec"
+# encoding: utf-8
+require "logstash/devutils/rake"
+require "jars/installer"
+require "fileutils"
 
-require 'jars/installer'
-desc 'Install the JAR dependencies to vendor/'
-task :install_jars do
-  # We actually want jar-dependencies will download the jars and place it in
-  # vendor/jar-dependencies/runtime-jars
-  Jars::Installer.new.vendor_jars!(false, 'vendor/jar-dependencies/runtime-jars')
+task :default do
+  system('rake -vT')
 end
 
-task build: :install_jars
-require "logstash/devutils/rake"
-task vendor: :install_jars
+task :vendor do
+  exit(1) unless system './gradlew vendor'
+end
 
+task :clean do
+  ["vendor/jar-dependencies", "Gemfile.lock"].each do |p|
```

Comment:
I may be missing some context, but why delete `Gemfile.lock` when cleaning? And why is the file in `.gitignore`?

The lockfile is important to share a very precise known good set of resolved dependencies between developers working on this...

If the issue is that we need to test on multiple incompatible configurations (e.g. JRuby 1.7 & 9.x) and a single Gemfile doesn't cut it, we can always use a the [appraisal](/thoughtbot/appraisal) gem to drive bundler.

Reply:
we don't have the habit of committing the .lock file in the plugins, so the .lock is listed in the .gitignore.

When a plugin is used with logstash, logstash won't care about any locked dependencies besides the ones in the gemspec. So from this perspective, I think that recomputing the .lock on every test run is more likely to catch dependency-related issues that we would see on logstash, which is a good thing.